.vscode/launch.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Please revert this.  I don't want to get into the business of having the config files for every users editor in my repos

Reply:
(in git, you can configure a user-wide ignore and place these here for yourself)